src/Features/RulesMissingDocumentation.md

```diff
@@ -4,13 +4,6 @@ Rule ID | Missing Help Link | Title |
 --------|-------------------|-------|
 IDE0005_gen |  | Imports statement is unnecessary. |
 IDE0043 |  | Invalid format string |
-IDE0240 | <https://learn.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0240> | Remove redundant nullable directive |
-IDE0241 | <https://learn.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0241> | Remove unnecessary nullable directive |
-IDE0250 | <https://learn.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0250> | Make struct 'readonly' |
-IDE0251 | <https://learn.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0251> | Make member 'readonly' |
-IDE0260 | <https://learn.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0260> | Use pattern matching |
-IDE0270 | <https://learn.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0270> | Use coalesce expression |
-IDE0280 | <https://learn.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0280> | Use 'nameof' |
 IDE0290 | <https://learn.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0290> | Use primary constructor |
```

Comment:
@gewarren FYI on https://developercommunity.visualstudio.com/t/Documentation-Missing-for-Code-Analyzer-/10475249